README.md

```diff
@@ -26,6 +26,15 @@ If you'd like your tests to run automatically when they change, we recommend the
 ```bash
 find . -name "*_shpec.sh" | entr shpec
 ```
+### Test structure
+`Shpec` would be very familiar if you already know *BDD*, and test framework such as `rspec`, `jasmine`, `mocha`, *et caetera*
```

Comment:
`shpec`'s syntax is very similar to other BDD frameworks such as `RSpec`, `jasmine`, or `mocha`. (add github links, please).